pom.xml

```diff
@@ -88,6 +94,16 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<artifactId>maven-jar-plugin</artifactId>
```

Comment:
I don't remember if adding this plugin config or updating the assembly plugin resolved the build problem.